generate-lighttpd-config.sh

```diff
@@ -22,23 +24,33 @@ server.groupname = "www"
 server.errorlog = "/dev/stderr"
 
 accesslog.filename = "/dev/fd/2"
+' > "$configfile"
 
-proxy.server = (
-  "" => (' > $configfile # overwrite current config file
-
-for i in $(seq 1 "$TREX_INSTANCES")
+for count in "${!TREX_COUNT@}"
```

Comment:
o wat is bash toch intuitief ;). dit kende ik nog niet. heb je een naam/linkje naar wat hier gebeurt?

Reply:
parameters expansion, zie https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html (zoek op `@`). Hier meer praktisch voorbeeld https://unix.stackexchange.com/a/460466. Ik zoek alle env vars die beginnen met een prefix https://unix.stackexchange.com/a/460466